components/uaepass-authenticator/src/main/java/org/wso2/carbon/identity/uaepass/authenticator/UAEPassAuthenticatorConstants.java

```diff
@@ -0,0 +1,120 @@
+            public static final String UAEPASS_STG_LOGOUT_ENDPOINT_VALUE = "https://stg-id.uaepass.ae/idshub/logout";
+        }
+
+        public class ProductionEndpointValues {
+            public static final String UAEPASS_PROD_AUTHZ_ENDPOINT_VALUE = "https://id.uaepass.ae/idshub/authorize";
+            public static final String UAEPASS_PROD_TOKEN_ENDPOINT_VALUE = "https://id.uaepass.ae/idshub/token";
+            public static final String UAEPASS_PROD_USER_INFO_ENDPOINT_VALUE = "https://id.uaepass.ae/idshub/userinfo";
+            public static final String UAEPASS_PROD_LOGOUT_ENDPOINT_VALUE = "https://id.uaepass.ae/idshub/logout";
+        }
+    }
+
+    public class UAE{
+        public static final String FEDERATED_IDP_COMPONENT_NAME = "UAEPassAuthenticator";
+        public static final String FEDERATED_IDP_COMPONENT_FRIENDLY_NAME = "UAEPass";
+        public static final String LOGIN_TYPE = "OIDC";
+        public static final String ACCESS_TOKEN = "access_token";
+        public static final String ID_TOKEN = "id_token";
+        public static final String CLIENT_ID = "client_id";
+        public static final String CLIENT_SECRET = "ClientSecret";
+        public static final String CALLBACK_URL = "callbackUrl";
+        public static final String ACR_VALUES = "acr_values";
+        public static final String RESPONSE_TYPE = "response_type";
+        public static final String REDIRECT_URI = "redirect_uri";
+        public static final String SCOPE = "scope";
+        public static final String OAUTH2_GRANT_TYPE_CODE = "code";
+        public static final String OAUTH2_PARAM_STATE = "state";
+        public static final String QUERY_PARAMS = "commonAuthQueryParams";
+        public static final String UAEPASS_ENV = "IsStagingEnv";
+        public static final String HTTP_ORIGIN_HEADER = "Origin";
+        public static final String LOGOUT_ENABLE = "IsLogoutEnable";
+
+    }
+
+    public class UAEPassPropertyConstants {
+        public static final String TEXTBOX = "string";
+        public static final String CHECKBOX = "boolean";
+    }
+
+    public static class UAEPassRuntimeConstants {
+        public static final String SUB = "sub";
+        public static final String OIDC_DIALECT = "http://wso2.org/oidc/claim";
+        public static final String DEFAULT_ACR_VALUES = "urn:safelayer:tws:policies:authentication:level:low";
+        public static final String DEFAULT_SCOPES = "urn:uae:digitalid:profile:general";
+        public static final String[] NON_USER_ATTRIBUTES = new String[]{"at_hash", "iss", "iat", "exp", "aud", "azp"};
+        public static final String PRODUCTION = "Production";
+        public static final String STAGING = "Staging";
+        public static final String SANDBOX_STAGE_CLIENT_ID = "sandbox_stage";
+    }
+
+    public enum ErrorMessages {
+
+        USER_ID_NOT_FOUND_IN_ID_TOKEN_SENT_BY_FEDERATED_IDP("OID-60005",
```

Comment:
Better to use an error code prefix that are relevant to the authenticator like UAE-60000